src/SFML/Window/VideoMode.cpp

```diff
@@ -60,7 +60,7 @@ const std::vector<VideoMode>& VideoMode::getFullscreenModes()
     static const auto modes = []
     {
         std::vector<VideoMode> result = priv::VideoModeImpl::getFullscreenModes();
-        std::sort(result.begin(), result.end(), std::greater<VideoMode>());
+        std::sort(result.begin(), result.end(), std::greater<>());
```

Comment:
I don't know std::sort or std::greater well enough, why can we leave out the template type?

Reply:
Honesty not sure how transparent functors work but apparently it's a thing we can do.